packages/cyberstorm/src/components/PackageList/PackageList.tsx

```diff
@@ -69,32 +63,44 @@ export function PackageList(props: Props) {
     deferredDeprecated,
   ]);
 
+  const isNotStale =
+    deferredOrder === order &&
+    deferredPage === page &&
+    deferredSearchQuery === searchQuery &&
+    deferredCategories === categories &&
+    deferredSection === section &&
+    deferredNsfw === nsfw &&
+    deferredDeprecated === deprecated;
+
   return (
     <div className={styles.root}>
```

Comment:
Nitpick: I would consider using the `StaleCover` over the whole component instead of individual subcomponents.

* It would be less repetition (not a huge problem here but can be on future components with more subcomponents, and it would be nice to future proof this be consistent site-wide)
* If we add some spinner icon oslt it will be easier to position e.g. to center of the area. This ofc is irrelevant if we won't add one.
* It would cover the PackageOrder - I personally can't decide if I like it more or less but it's a matter of opinion I think

Reply:
> `PackageOrder` has to be available, even if the other stuff is loading.

Does it though?